### DISCUSSION TITLE
SCreenshot key name ?

### DISCUSSION THREAD

**Discussion by NicTanghe** [leftwm/leftwm#1082] [Q&A]
Hello I`ve been trying to find the name of the screenshot key so i can bind it to gnome-screenshot.
But xev comes up empty so I`m a bit confused on how to preoceed.

**Comment by VuiMuich** [comment]
Maybe have a look here: https://askubuntu.com/questions/1197651/ubuntu-show-what-keys-are-pressed-in-real-time

Also a Little more info on your keyboard might be helpful, as `screenshot` key is not something common ime.
